docs/content/installation/upgrade-from-gitea.en-us.md

```diff
@@ -60,6 +64,10 @@ Backup steps:
 If you are using cloud services or filesystems with snapshot feature,
 a snapshot for the Gitea data volume and related object storage is more convenient.
 
+After all of those have been prepared, to update Gitea, download a newer version, stop the old one, perform a backup, and run the new one.
+Every time a Gitea instance starts up, it checks whether a database migration should be run.
+If a database migration is required, Gitea will take some time to complete the upgrade and then serve.
```

Comment:
```suggestion
After all of steps have been prepared, download the new version, stop the application, perform a backup and
then start the new application. On each startup, Gitea verifies that the database is up to date and will automtically
perform any necessary migrations. Depending on the size of the database, this can take some additional time on the
first launch during which the application will be unavailable.
```